.devcontainer/scripts/setup-local-settings.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+set -e
+
+
+cp ./source/workflow-designtime/starter.settings.json ./source/workflow-designtime/local.settings.json
```

Comment:
Also, how about `local.template.settings.json` or similar that links it to the `local.settings.json` a bit more strongly?